src/foam/java/refinements.js

```diff
@@ -518,7 +539,9 @@ foam.CLASS({
       getter: function() {
         // TODO: This could be an expression if the copyFrom in createChildMethod
         // didn't finalize its value
-        if ( this.name == 'find' ) console.log(this.name, "returns", this.javaReturns)
+        if ( this.name == 'find' ) {
```

Comment:
Why?

Reply:
The string was longer than 80 characters.